transport/graphsync/graphsync.go

```diff
@@ -1103,6 +1107,35 @@ func (c *dtChannel) gsReqOpened(gsKey graphsyncKey, hookActions graphsync.Outgoi
 	c.opened <- gsKey
 }
 
+func (c *dtChannel) gsDataRequestQueued(gsKey graphsyncKey, hookActions graphsync.IncomingRequestHookActions) {
```

Comment:
Is this method being called from somewhere?

Reply:
ha oops forgot i added that